react/components/ProductAvailabilityWrapper.tsx

```diff
@@ -131,50 +93,65 @@ function ProductAvailabilityWrapper({
     <CssHandlesProvider handles={handles} withModifiers={withModifiers}>
       <ProductAvailability
         threshold={threshold}
-        lowStockMessage={lowStockMessage}
-        highStockMessage={highStockMessage}
+        lowStockMessage={formattedLowStockMessage}
+        highStockMessage={formattedHighStockMessage}
         showAvailability={showAvailability}
-        showAvailabilityMessage={showAvailabilityMessage}
+        showAvailabilityMessage={formattedShowAvailabilityMessage}
         availableQuantity={availableQuantity}
       />
     </CssHandlesProvider>
   )
 }
 
-ProductAvailabilityWrapper.schema = {
-  title: messages.title.id,
-  description: messages.description.id,
-  type: 'object',
-  properties: {
-    threshold: {
-      title: messages.thresholdTitle.id,
-      description: messages.thresholdDescription.id,
-      type: 'number',
-      default: 0,
-      isLayout: true,
-    },
-    lowStockMessage: {
-      title: messages.lowStockMessageTitle.id,
-      description: messages.lowStockMessageDescription.id,
-      type: 'string',
-    },
-    highStockMessage: {
-      title: messages.highStockMessageTitle.id,
-      description: messages.highStockMessageDescription.id,
-      type: 'string',
-    },
-    showAvailability: {
-      title: messages.showAvailabilityTitle.id,
-      description: messages.showAvailabilityDescription.id,
-      type: 'enum',
-      enum: ['disabled', 'stock'],
-    },
-    showAvailabilityMessage: {
-      title: messages.showAvailabilityMessageTitle.id,
-      description: messages.showAvailabilityMessageDescription.id,
-      type: 'string',
-    },
+defineMessages({
+  title: {
+    defaultMessage: 'Product Availability',
+    id: 'admin/editor.product-availability.title',
+  },
+  lowStockMessageTitle: {
+    defaultMessage: 'Product Availability',
```

Comment:
The following default messages are how they should be or did you forget to update them?

Reply:
Hehe, forgot to update.